### ISSUE TITLE
send action bar doesn't work

### ISSUE THREAD

**Issue by nshun1896** [TheLimeGlass/Skellett#36]
Thanks for the 1.9.2 update.
I found the problem with send action bar.

The error log is bellow..
https://pastebin.com/cCNBtMfF

Spigot and plugins I used are bellow.
```
[22:02:45] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-60095c9-a479029 (MC: 1.12-pre5) (Implementing API version 1.12-pre5-SNAPSHOT)
[22:02:46] [Server thread/INFO]: [Skript] Loading Skript v2.2-dev27
[22:02:46] [Server thread/INFO]: [Skellett] Loading Skellett v1.9.2
[22:02:46] [Server thread/INFO]: [SkQuery] Loading SkQuery v3.5.6-Lime
```

**Comment by TheLimeGlass** [comment]
Interesting. It seems Mojang has removed the IChatBaseComponent with byte option for 1.12+ I will check it out and fix it sooner or later.

Does this also happen with Titles?

I can say alot of plugins are going to be broken with this 1.12 version if actionbars are broken lol

**Comment by nshun1896** [comment]
Thanks so much!

I use skript's function to display title, but I tested the following code now, and then it seems not work.
`show a title from "&6Title" and subtitle "&aSubtitle" to player for 5 seconds and 10 ticks fade in and 15 ticks fadeout`

and also tested sound,
`play sound "entity_ghast_shoot" to player with volume 1 and pitch 1`

It worked fine, but seems very low sound and it took long time reloading the code above. 
"1" is default pitch right?

**Comment by TheLimeGlass** [comment]
The sound effect uses these sounds https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Sound.html

And the volume goes between 0 and 1 and the pitch goes between 1 and 15

**Comment by TheLimeGlass** [comment]
Fixed in 1.9.4 https://github.com/TheLimeGlass/Skellett/releases

Will work on titles soon.
